addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -164,6 +164,7 @@ class MessageList extends Component {
                 threadView.markComponentHintProcessed(hint);
             }
         }
+        this.scrollableParent = this.getScrollableParent(this.el);
```

Comment:
First of all: the value returned by this function should never be cached but always called explicitly, because at any time the parent chain could have changed and the element that would be returned could be different.